src/app/carbon-estimation/carbon-estimation.component.spec.ts

```diff
@@ -47,32 +47,26 @@ describe('CarbonEstimationComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should set chart height when inner height is more than base height', () => {
+  it('should set the chart height when the component is initialised', () => {
+    spyOnProperty(window, 'innerHeight').and.returnValue(1000);
+    spyOnProperty(window, 'innerWidth').and.returnValue(1000);
+    spyOnProperty(window.screen, 'height').and.returnValue(1080);
     spyOnProperty(component.detailsPanel.nativeElement, 'clientHeight').and.returnValue(200);
-    component.ngOnInit();
-    fixture.detectChanges();
 
-    expect(component.chartOptions.chart.height).toBe(600 - estimatorBaseHeight - 200);
-  });
-
-  it('should set chart height to value if inner height more than base height plus extra height', () => {
-    spyOnProperty(component.detailsPanel.nativeElement, 'clientHeight').and.returnValue(200);
-    fixture.componentRef.setInput('extraHeight', '100');
```

Comment:
Looks like we've lost all tests around setting the extraHeight input, I think we want to have at least one around this, even if it's changed a bit.

Reply:
I've added a test for the extraHeight input.